OPAL/br/src/main/scala/org/opalj/br/fpcf/properties/string_definition/StringConstancyLevel.scala

```diff
@@ -0,0 +1,76 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.br.fpcf.properties.string_definition
+
+/**
+ * Values in this enumeration represent the granularity of used strings.
+ *
+ * @author Patrick Mell
+ */
+object StringConstancyLevel extends Enumeration {
```

Comment:
is there a reason for not using case objects?

Reply:
No. Resolved with 586821a0b04138879c44c90476ed8c01a19ebb72.